files/default/neutron-ha-tool.py

```diff
@@ -104,14 +107,19 @@ def parse_args():
                          'is mainly useful when evacuating routers from '
                          'a node where the l3-agent is not running for '
                          'some reason.')
+    ap.add_argument('--agent-selection-mode', choices=['random', 'least-busy'],
+                    default='least-busy',
+                    help='Determines how target agent is selected for routers '
+                         '"random" selects target agent randomly, '
+                         '"least-busy" selects the least busy agent.')
```

Comment:
It sounds to me like the output of this may not be the three lines you have here -- couldn't it happen that the first two lines are merged, which will result in some not so nice text?